### PR TITLE
ops: setup dependabot

### PR DESCRIPTION
**Pull request by ptrus**
Fixes: https://github.com/oasisprotocol/oasis-sdk/issues/79

Hopefully the spam wont be to bad :grin:

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    directory: client-sdk/ts-web
+    schedule:
+      interval: daily
+      # Check for updates at 7am UTC.
+      time: "07:00"
+    commit-message:
+      prefix: "ts-web:"
+    labels:
+      - c:deps
+      - typescript
```
Comment (kostko) [inline .github/dependabot.yml:60]:
```suggestion
      - c:client-sdk/typescript
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      time: "07:00"
+    commit-message:
+      prefix: "ci:"
+    labels:
+      - c:deps
+      - c:testing
+
+  # Manage Go package versions.
+  - package-ecosystem: gomod
+    directory: client-sdk/go/
```
[the remark this answers is not in the document]
Comment (ptrus) [reply .github/dependabot.yml:17]: added a configuration for tests/e2e as well

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  # Manage npm package versions.
+  - package-ecosystem: npm
+    directory: client-sdk/ts-web
+    schedule:
+      interval: daily
+      # Check for updates at 7am UTC.
+      time: "07:00"
+    commit-message:
+      prefix: "ts-web:"
```
[the remark this answers is not in the document]
Comment (ptrus) [reply .github/dependabot.yml]: Apparently 15 chars is the max `prefix` length

[the remark this answers is not in the document]
Comment (ptrus) [reply .github/dependabot.yml]: Could do `client/ts-web:`

**Pull request merged**